### PR TITLE
Add Casamarciano station

### PR DESCRIPTION
**Pull request by Dzyunyakl**
Add Casamarciano and Palermo De Gasperi stations

### CODE REVIEW

File: stations.csv [hunk 1]
```diff
@@ -3202,6 +3202,8 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 4194;Biot;biot;8775769;87757690;43.610951;7.126090;;FR;Europe/Paris;f;f;f;t;f;f;FRIBK;BJO;t;;f;8700723;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv4194;
 4195;Grasse;grasse;8775772;87757724;43.653317;6.925524;;FR;Europe/Paris;f;f;f;t;f;f;FRIBL;GRA;t;;f;8705486;f;;f;;f;;f;;f;;f;8705486;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;グラース;그라스;;;Грас;;;格拉斯;urn:trainline:public:nloc:csv4195;
 4196;Ranguin;ranguin;8775773;87757732;43.5691636;6.9693134;;FR;Europe/Paris;f;f;f;t;f;f;FRIBM;RGN;t;;f;8704345;f;;f;;f;;f;;f;;f;8704345;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv4196;
+4222;Casamarciano;casamarciano;;;40.93125823842119;14.548592083542841;;IT;Europe/Paris;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv4222;
```
Comment (ConscritNeuneu) [inline stations.csv:3205]: Never reuse ids. Who knows what this 4222 represented in the past. New stations must be added at the end of the file.

Comment (ConscritNeuneu) [reply stations.csv:3205]: Also, which station is this? It doesn't have any vendor code and there seems to be no train station passing close.
openstreetmap

https://www.openstreetmap.org/#map=15/40.9313/14.5486&layers=T

**Comment by ConscritNeuneu** [comment]
Ah, finally, please avoid referencing internal Trainline jira tickets as it's a public repo.

File: stations.csv (hunk 1 above)
Comment (Dzyunyakl) [reply stations.csv:3205]: Looks like it's a bus station
https://maps.app.goo.gl/gE6tHU52M3uDM9347

Comment (Dzyunyakl) [reply stations.csv:3205]: Not sure what vendor it should use tho

File: stations.csv [hunk 2]
```diff
@@ -70515,3 +70515,4 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 74661;Firenze Piazzale Montelungo;firenze-piazzale-montelungo;;;43.780650;11.246700;8433;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;@ITFLRPMO;t;24648;t;;f;;f;;f;;f;8314717;t;;;;;f;;f;;f;;f;;f;;f;f;;Florenz;Florence;Florencia;Florence;;;Florence;Florencie;;;フィレンツェ;피렌체;Firence;Florença;Флоренция;Florens;Floransa;佛罗伦萨;urn:trainline:public:nloc:csv74661;
 74662;Senigallia Lungomare Sud;senigallia-lungomare-sud;;;43.703090;13.243700;20267;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8314768;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;セニガッリア;;;;Сенигаллия;;;;urn:trainline:public:nloc:csv74662;
 74663;Palermo De Gasperi;palermo-de-gasperi;;;38.1480195;13.3285637;22186;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8312065;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;Palerme;;;;;;;パレルモ;팔레르모;;;Палермо;;;巴勒莫市;urn:trainline:public:nloc:csv74663;
+74664;Casamarciano;casamarciano;;;40.93125823842119;14.548592083542841;;IT;Europe/Paris;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74664;
```
Comment (misdoro) [inline stations.csv:70518]: :thinking: Where are the coordinates coming from? Can you pls trim them down to 6 digits after the `.` max?

Comment (ConscritNeuneu) [reply stations.csv:70518]: We should have a test for that :sparkles: :smiling_imp:

Comment (Dzyunyakl) [reply stations.csv:70518]: Thank you! Trimmed it to 6 digits after the `.`

**Review by ConscritNeuneu — APPROVED** [review]

**Pull request merged**